.github/workflows/dotnetcore.yml

```diff
@@ -25,6 +25,10 @@ jobs:
         uses: actions/setup-dotnet@v3.0.3
         with:
           dotnet-version: '6.x'
+      - name: Setup dotnet 7
```

Comment:
:wrench:  please take the change and change this to the 
"multi version syntax" as described here https://github.com/actions/setup-dotnet#usage

```yaml
steps:
- uses: actions/checkout@v3
- name: Setup dotnet
  uses: actions/setup-dotnet@v3
  with:
    dotnet-version: | 
      3.1
      3.1.x
      7.x
```

Reply:
@JanKrivanek  I can change this too  would this also apply to .github/workflows/release.yml
![image](https://user-images.githubusercontent.com/6450533/217797669-5fa7bb26-bfd0-4616-bd0f-d81dc59b3cc4.png)

Reply:
@jkowalleck  Have made these changes to the yml files

Reply:
i do not know if the release process needs all .net versions installed to do the job. 
you decide.